packages/react-card/Spec.md

```diff
@@ -0,0 +1,403 @@
+| Component to render an image                                                                                               | DocumentCardImage    | CardPreview       | ⚠️        |
+| Hybrid component to render an image preview for a document or list of documents. Possibly redundant with DocumentCardImage | DocumentCardPreview  |                   | ❌        |
+| Adds an icon/logo to the card                                                                                              | DocumentCardLogo     |                   | ❌        |
+| Wrapper for the right side of horizontal cards                                                                             | DocumentCardDetails  |                   | ❌        |
+| Show document activity (user list)                                                                                         | DocumentCardActivity |                   | ❌        |
+| Link element                                                                                                               | DocumentCardLocation |                   | ❌        |
+| Extra text component with a grayed out background                                                                          | DocumentCardStatus   |                   | ❌        |
+| Header of the card                                                                                                         |                      | CardHeader        | ❌        |
+| Body of the card                                                                                                           |                      | CardBody          | ❌        |
+| Footer of the card                                                                                                         |                      | CardFooter        | ❌        |
+| Wrapper to render in column format                                                                                         |                      | CardColumn        | ❌        |
+| Allows for content to expand on hover                                                                                      |                      | CardExpandableBox | ❌        |
+
+...
+
+## API Proposal
+
+### Components
+
+| Component   | Purpose                                                                  |
+| ----------- | ------------------------------------------------------------------------ |
+| Card        | Wrapper for the main content of a card.                                  |
+| CardHeader  | Optional header slot for the card.                                       |
+| CardBody    | Main slot of the Card                                                    |
+| CardPreview | Previewing the document/article presented in the card through an image   |
+| CardFooter  | Optional footer slot for the card. (non-applicable for horizontal cards) |
+
```

Comment:
can you please add a section for cards container? It does not necessarily have to be a separate component, but we should at least have documentation about how to group several cards and what interactions are possible.

Cards are usually laid out in a grid, so we need to decide how should we represent that grid from accessibility point of view. It might be challenging as there are no good roles that would represent this - `menu` and `list`/`listbox` are linear, `grid` has strict DOM structure requirements that usually does not play well with layout and also brings narration of rows/columns that might be confusing. 

As for navigation between the cards, we should look into tabster support for 2 dimensional grids and the ability to enter/escape to/from card. Alternative would be to use tab to go inside of the card. 

Probably somehing to discuss separately, but since cards are in a container most of the time, it is an important topic.

Reply:
I understand the problem but think it should definitely be covered elsewhere. Adding that would add to the complexity of this Spec as it's now also covering layout scenarios of cards.
While important, I think it should either be a component of itself or documented/defined elsewhere as the focus is different from a Card implementation.
What do you think?